tests/requirements.txt

```diff
@@ -2,9 +2,9 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file=tests/requirements.txt tests/requirements.in
+#    pip-compile
 #
-amqp==5.0.2               # via kombu
```

Comment:
@jparsai This file `requirements.txt` is auto generate by command 
`pip-compile --output-file=requirements.txt requirements.in` 

It is recommended not to modify this file directly. If you want, you can version lock in `requirement.in` itself

Reply:
@deepak1725 Yes I have not updated it manually, I used pip-compile only, it is modified because main requiremnts.in file is updated and it is inherited by test/requirements.in so if test/requirements.in is not modified still test/requirements.txt will have changes due to parent requirements.in

Reply:
Okay